src/components/ItemsNotFoundComponent.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { Image } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+import '../_css/ItemsNotFoundComponent.css';
```

Comment:
Do you really need to import the style file here? What happens if you don't do it? The expectation is that all assets should be compiled in one file, containing all the classes so the import is unnecessary